NOTICE.txt

```diff
@@ -62,6 +62,7 @@ This product contains a modified portion of 'Apache Harmony', an open source
 Java SE, which can be obtained at:
 
   * LICENSE:
```

Comment:
Would be more accurate to put it under a NOTICE: header, but this is fine too.

Reply:
Sure. Will update.

Reply:
yeah but it's still in the license directory.

Reply:
Yeah, but I'm not sure I want to add another top-level directory just for this. Perhaps when we have more of these?